src/components/VerticalNavigation/VerticalNavigationBrand.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const VerticalNavigationBrand = props => {
+  const { title, href, iconImg, children } = props;
+  // The img prop is just a shorthand for the titleImg prop.
+  // When also using iconImg, it is less confusing to pass titleImg instead of img.
+  const titleImg = props.titleImg || props.img;
+
+  const brandChildren = children || (
+    <span>
+      {iconImg && (
+        <img className="navbar-brand-icon" src={iconImg} alt={title} />
+      )}
+      {titleImg && (
+        <img className="navbar-brand-name" src={titleImg} alt={title} />
+      )}
+      {!titleImg && title && <span className="navbar-brand-txt">{title}</span>}
+    </span>
+  );
+  return href ? (
+    <a href={href} className="navbar-brand">
+      {brandChildren}
+    </a>
+  ) : (
+    <span className="navbar-brand">{brandChildren}</span>
+  );
+};
+
+VerticalNavigationBrand.propTypes = {
```

Comment:
Would be useful to add storybook descriptions.

Reply:
Definitely on the short list of remaining things.  And as for that little bit of CSS, maybe it should be in pf core but I wasn't sure where that style was coming from in patternfly-ng, it appears to render that way but I didn't see that CSS anywhere. Still needs a little investigating.